pkg/build.ml

```diff
@@ -6,6 +6,8 @@ let () =
   Pkg.describe "protobuf" ~builder:`OCamlbuild [
     Pkg.lib "pkg/META";
     Pkg.lib ~exts:Exts.module_library "src/protobuf_codec";
+    Pkg.lib ~exts:[".ml"]     "src_test/test_protobuf_codec";
+    Pkg.lib ~exts:[".native"] "src_test/test_protobuf";
```

Comment:
Somehow `make test` was not working for me. This PR is first to highlight the problem and discuss what should be the solution, I'll clean everything!

Did you manage to reproduce the issue and agree it's in fact not the expected behavior ?